packages/xchain-thorchain/package.json

```diff
@@ -27,29 +27,36 @@
     "clean": "rimraf lib/**",
     "build": "yarn clean && rollup -c",
     "test": "jest",
+    "e2e": "jest --config jest.config.e2e.js",
     "lint": "eslint \"{src,__tests__}/**/*.ts\" --fix --max-warnings 0",
     "prepublishOnly": "yarn build",
-    "start:example": "ts-node example/index.ts"
+    "start:example": "ts-node example/index.ts",
+    "generate:ThorchainMsgs": "./genMsgs.sh"
   },
   "devDependencies": {
+    "@cosmos-client/core": "^0.44.4",
     "@types/big.js": "^6.0.0",
     "@xchainjs/xchain-client": "^0.11.1",
-    "@xchainjs/xchain-cosmos": "^0.16.1",
+    "@xchainjs/xchain-cosmos": "^0.17.0",
     "@xchainjs/xchain-crypto": "^0.2.4",
     "@xchainjs/xchain-util": "^0.5.1",
     "axios": "^0.25.0",
-    "cosmos-client": "0.39.2",
+    "bech32-buffer": "^0.2.0",
     "nock": "^13.0.5"
   },
   "publishConfig": {
     "access": "public"
   },
   "peerDependencies": {
+    "@cosmos-client/core": "^0.44.4",
     "@xchainjs/xchain-client": "^0.11.1",
-    "@xchainjs/xchain-cosmos": "^0.16.1",
+    "@xchainjs/xchain-cosmos": "^0.17.0",
     "@xchainjs/xchain-crypto": "^0.2.4",
     "@xchainjs/xchain-util": "^0.5.1",
     "axios": "^0.25.0",
-    "cosmos-client": "0.39.2"
+    "bech32-buffer": "^0.2.0"
+  },
+  "dependencies": {
```

Comment:
Remove `bech32-buffer` from `dependencies` - it's already in `devDependencies` + `peerDependencies`